build.gradle

```diff
@@ -11,6 +11,7 @@ plugins {
     id 'net.minecraftforge.gradle.mcp' version '[6.0.22,6.2)' apply false
     id 'net.minecraftforge.gradlejarsigner' version '1.0.4'
     id 'org.barfuin.gradle.taskinfo' version '2.1.0'
+    id 'net.minecraftforge.gradle' version '5.1+'
```

Comment:
No need to add this. You should never need to change the buildscript of the project unless you need to edit the dependencies in some way, which doesn't seem to be the case here.